src/schedule/ScheduleCreateTransaction.js

```diff
@@ -298,6 +312,41 @@ export default class ScheduleCreateTransaction extends Transaction {
         );
         return `ScheduleCreateTransaction:${timestamp.toString()}`;
     }
+
+    /**
+     * @param {?Timestamp} expirationTime
+     * @returns {this}
+     */
+    _setExpirationTime(expirationTime) {
+        this._expirationTime = expirationTime;
+        return this;
+    }
+
+    /**
+     * @returns {?Timestamp}
+     */
+    get expirationTime() {
+        this._requireNotFrozen();
+        return this._expirationTime;
+    }
+
+    /**
+     * @param {boolean} waitForQuery
+     * @returns {this}
+     */
+    _setWaitForQuery(waitForQuery) {
+        this._waitForQuery = waitForQuery;
+
+        return this;
+    }
+
+    /**
+     * @returns {?boolean}
+     */
+    get waitForQuery() {
+        this._requireNotFrozen();
+        return this._waitForQuery;
+    }
```

Comment:
I think you need to update the `[to|from]Protobuf()` methods to support  this new field.

Reply:
updated to include field.